crates/bitwarden/src/tool/generators/passphrase.rs

```diff
@@ -10,28 +13,51 @@ use serde::{Deserialize, Serialize};
 #[serde(rename_all = "camelCase", deny_unknown_fields)]
 #[cfg_attr(feature = "mobile", derive(uniffi::Record))]
 pub struct PassphraseGeneratorRequest {
+    /// Number of words in the generated passphrase.
+    /// This value must be between 3 and 20.
+    /// The default value when unset is 3
     pub num_words: Option<u8>,
+    /// Character separator between words in the generated passphrase.
+    /// If the value is set, it cannot be empty.
+    /// The default value when unset is `-`
     pub word_separator: Option<String>,
+    /// When set to true, capitalize the first letter of each word in the generated passphrase.
+    /// The default value when unset is `false`
     pub capitalize: Option<bool>,
+    /// When set to true, include a number at the end of one of the words in the generated passphrase.
+    /// The default value when unset is `false`
     pub include_number: Option<bool>,
 }
 
 const DEFAULT_PASSPHRASE_NUM_WORDS: u8 = 3;
-const DEFAULT_PASSPHRASE_SEPARATOR: char = ' ';
+const DEFAULT_PASSPHRASE_SEPARATOR: &str = " ";
+
+const MINIMUM_PASSPHRASE_NUM_WORDS: u8 = 3;
+const MAXIMUM_PASSPHRASE_NUM_WORDS: u8 = 20;
 
 pub(super) fn passphrase(input: PassphraseGeneratorRequest) -> Result<String> {
     passphrase_with_rng(rand::thread_rng(), input)
 }
```

Comment:
⚠️ Here's another undocumented public member. I'm not gonna point them all out, but they should all be documented--at least within tools-owned code.

Reply:
I don't follow what you're saying. Could you point to the code you're talking about?

Reply:
The public API function is here: 
https://github.com/bitwarden/sdk/blob/8d54b988deda55fd7af5c369db308b7428c0c747/crates/bitwarden/src/tool/generators/client_generator.rs#L17-L38

Which just calls the function we are talking about, which is pub(super), so only accessible within the generators module.

This is the pattern we've been following to avoid having a super-long client_generator file.

Reply:
⚠️ Cool cool. Please link from the inner `passphrase(...)` to this one using a [name link](https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html#linking-to-items-by-name).

Reply:
My bad. I missed that. Thanks! :)